test/tests/meshgenerators/block_deletion_generator/block_deletion_test5.i

```diff
@@ -26,7 +26,7 @@
   [../]
   [./ed0]
     type = BlockDeletionGenerator
-    block = 1
+    block_id = 1
```

Comment:
Did you make this change in order to preserve coverage? 

I would rename this test since you deleted tests 2, 3, and 4

Reply:
Yep, the coverage was lost on the block_id deprecated param, and yeah, the test can be renamed